From b0ba6a1dee995b3cd7331b4df92d115e51d8ac0c Mon Sep 17 00:00:00 2001
From: Jes Sorensen <Jes.Sorensen@gmail.com>
Date: Wed, 29 Mar 2017 15:48:24 -0400
Subject: [RHEL7.5 PATCH 041/169] md_u: Remove some unused ioctl
 declarations

These were no longer used in the code, so get rid of them.

Signed-off-by: Jes Sorensen <Jes.Sorensen@gmail.com>
---
 md_u.h | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/md_u.h b/md_u.h
index f570a34..d59aa2d 100644
--- a/md_u.h
+++ b/md_u.h
@@ -21,19 +21,13 @@
 #define RAID_VERSION		_IOR (MD_MAJOR, 0x10, mdu_version_t)
 #define GET_ARRAY_INFO		_IOR (MD_MAJOR, 0x11, mdu_array_info_t)
 #define GET_DISK_INFO		_IOR (MD_MAJOR, 0x12, mdu_disk_info_t)
-#define PRINT_RAID_DEBUG	_IO (MD_MAJOR, 0x13)
 #define RAID_AUTORUN		_IO (MD_MAJOR, 0x14)
 #define GET_BITMAP_FILE		_IOR (MD_MAJOR, 0x15, mdu_bitmap_file_t)
 
 /* configuration */
-#define CLEAR_ARRAY		_IO (MD_MAJOR, 0x20)
 #define ADD_NEW_DISK		_IOW (MD_MAJOR, 0x21, mdu_disk_info_t)
 #define HOT_REMOVE_DISK		_IO (MD_MAJOR, 0x22)
 #define SET_ARRAY_INFO		_IOW (MD_MAJOR, 0x23, mdu_array_info_t)
-#define SET_DISK_INFO		_IO (MD_MAJOR, 0x24)
-#define WRITE_RAID_INFO		_IO (MD_MAJOR, 0x25)
-#define UNPROTECT_ARRAY		_IO (MD_MAJOR, 0x26)
-#define PROTECT_ARRAY		_IO (MD_MAJOR, 0x27)
 #define HOT_ADD_DISK		_IO (MD_MAJOR, 0x28)
 #define SET_DISK_FAULTY		_IO (MD_MAJOR, 0x29)
 #define SET_BITMAP_FILE		_IOW (MD_MAJOR, 0x2b, int)
-- 
2.7.4

